plugin/cassandra/plugin.go

```diff
@@ -285,6 +365,25 @@ func (p CassandraPlugin) Validate(endpoint plugin.ShieldEndpoint) error {
 	return nil
 }
 
+func computeSavedKeyspaces(includeKeyspaces, excludeKeyspaces []string) []string {
+	if includeKeyspaces == nil {
+		return nil
+	}
+	savedKeyspaces := []string{}
+
+	sort.Strings(excludeKeyspaces)
+	for _, keyspace := range includeKeyspaces {
+		idx := sort.SearchStrings(excludeKeyspaces, keyspace)
+		if idx < len(excludeKeyspaces) && excludeKeyspaces[idx] == keyspace {
+			continue
+		}
+		savedKeyspaces = append(savedKeyspaces, keyspace)
+	}
+	sort.Strings(savedKeyspaces)
+
+	return savedKeyspaces
+}
```

Comment:
Can we remove some of the stutter in this code?  The work "Keyspace" appears often.  I believe `includeKeyspaces` could just be `include`, for example.